tests/test_dataset.py

```diff
@@ -73,7 +73,10 @@ def test_spatial_dataset(self):
             model_kwargs = json.load(file)
 
         model = Presto.construct(**model_kwargs)
+        # valid_month token can only be used in finetuning model
```

Comment:
@cbutsko what do you mean by this comment?

Reply:
my understanding is that currently in decoder there's no place for _valid_month_ token. so when I try to set valid_month_as_token=True to the whole Presto model in this fashion:
``` 
model = Presto.construct(**model_kwargs)
model.encoder.valid_month_as_token = True
eo, dw, mask, latlons, months, _, valid_months, _, _ = ds[0]

with torch.no_grad():
    _ = model(
        x=torch.from_numpy(eo).float()[:num_vals],
        dynamic_world=torch.from_numpy(dw).long()[:num_vals],
        latlons=torch.from_numpy(latlons).float()[:num_vals],
        mask=torch.from_numpy(mask).int()[:num_vals],
        month=torch.from_numpy(months).long()[:num_vals],
        valid_month=torch.from_numpy(valid_months).long()[:num_vals],
    )
```
I get the following error:
`RuntimeError: The size of tensor a (111) must match the size of tensor b (110) at non-singleton dimension 1`

which then leads me to this line in Decoder https://github.com/WorldCereal/presto-worldcereal/blob/6b09d830fdf033dedbdea13bd30c6865e8d04e16/presto/presto.py#L716

since we chose not to use valid_month in SSL now, I did not proceed into investigating this further and just made sure that encoder or finetuned model can use valid_month correctly.

Reply:
oh ok makes sense!